tests/Api/RepoTest.php

```diff
@@ -259,3 +259,15 @@
 
     expect($api->transfer('owenvoke', 'gitea-php', 'other-account', ['1234', '4321']))->toBe($expectedArray);
 });
+
+it("should fetch a raw file's data", function () {
+    $expectedArray = ['# Gitea PHP'];
+
+    $api = $this->getApiMock();
+    $api->expects($this->once())
+        ->method('get')
+        ->with('/repos/owenvoke/gitea-php/raw/README.md')
+        ->willReturn($expectedArray);
+
+    expect($api->show('owenvoke', 'gitea-php'))->toBe($expectedArray);
+});
```

Comment:
I've tested these changes to your test locally and it's working. 👍🏻
```suggestion
it("should fetch a raw file's data", function () {
    $expectedResponse = '# Gitea PHP';

    $api = $this->getApiMock();
    $api->expects($this->once())
        ->method('get')
        ->with('/repos/owenvoke/gitea-php/raw/README.md')
        ->willReturn($expectedResponse);

    expect($api->getRawFile('owenvoke', 'gitea-php', 'README.md'))->toBe($expectedResponse);
});
```

Reply:
Actually, I'll merge and then apply the changes separately. 👍🏻 Thanks for your work on this.